.ado/ado-packager-setup_macos.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Can eliminate a good amount of duplication by having ado-packager-setup_ios/macos call into a common help with a "platform" argument

Reply:
Yea, I was still learning all the syntactic sugar to get it running. For this PR I'm pausing on the CI packager to unblock getting the proj's in